101-traffic-manager-virtual-machines/azuredeploy.parameters.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+    "dnsname": {
+        "value": "somethingunique"
```

Comment:
For these unique values, please replace with "#####" to enable automated testing